sources/Externals/Soundfont/SFREADER.CPP

```diff
@@ -934,14 +958,23 @@ sfVersion *ver;
 
    resetInfoCkValues();   /* Make sure we are playing with a clean slate */
 
+   
+    printf ("infoCkValues\n");
    for(curToken = Ifil; curToken <= Icmt; curToken++) { 
-
+    printf ("  %d, %d, %d\n", curToken, tRIFF.GetRIFFToken(curToken), tRIFF.FindCk(tRIFF.GetRIFFToken(curToken)));
        if( tRIFF.FindCk(tRIFF.GetRIFFToken(curToken)) == SUCCESS ) {
-
-	 infoCkValues[curToken ] = new CHAR[ tRIFF.GetCkSize() + 1];
-
+     printf("  new CHAR %d\n", tRIFF.GetCkSize());
+	 
+     // SL: IssueHere - This allocation fails, the test below ends up being NULL
```

Comment:
The following check is failing suggesting that this allocation is failing. It might be related to the next comment.